common/at_client/test/u_at_client_test_data.c

```diff
@@ -869,15 +871,15 @@ static int32_t handleReadOnError(uAtClientHandle_t atClientHandle,
 {
     int32_t lastError;
     uint64_t uint64;
-    int64_t startTime;
-    int32_t duration;
+    uTimeoutStart_t timeoutStart;
```

Comment:
why not use uTimeoutStop_t and uTimeoutExpiredMs functions ?

Reply:
Yes, I guess we can lose the `duration`.